src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupsTable.jsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
```

Comment:
File comments

Reply:
Updated in 3c4bc853f806f3dad3df2644e1d880f41498bf3e